compose/material3/material3/src/skikoMain/kotlin/androidx/compose/material3/KotlinxDatetimeCalendarModel.kt

```diff
@@ -123,7 +123,9 @@ internal class KotlinxDatetimeCalendarModel(locale: CalendarLocale) : CalendarMo
     }
 
     override fun parse(date: String, pattern: String): CalendarDate? {
-        return platformDateFormat.parse(date, pattern)
+        return kotlin.runCatching {
```

Comment:
It looks like parsing exceptions handling is present in the implementations of `platformDateFormat` (For example: https://github.com/JetBrains/compose-multiplatform-core/blob/797ab9e20fb33920d600226324cc293d97561d3b/compose/material3/material3/src/jvmMain/kotlin/androidx/compose/material3/LegacyCalendarModelImpl.jvm.kt#L148). 

So it would make it wrapped in try/catch twice.
What do you think about moving this try/catch to web implementation? (webCommonW3C sourceset). 
So it would make it conform its behaviour to the implementations on other platforms.

Reply:
Sounds reasonable. Done